formats/format_gsm.c

```diff
@@ -174,6 +174,7 @@ static off_t gsm_tell(struct ast_filestream *fs)
 static struct ast_format_def gsm_f = {
 	.name = "gsm",
 	.exts = "gsm",
+	.mime_types= "audio/gsm|audio/x-gsm",
```

Comment:
There should be a space before the `=`.

Also, [`audio/gsm` is a registered mime type](https://www.iana.org/assignments/media-types/audio/GSM), so the `x-gsm` variant should not be necessary here.